Makefile

```diff
@@ -264,10 +264,11 @@ e2e_symfony:		 ## Packages a fresh Symfony app
 e2e_symfony: fixtures/build/dir012/vendor box
 	composer dump-env prod --working-dir fixtures/build/dir012
 
-	APP_ENV=prod APP_DEBUG=0 php fixtures/build/dir012/bin/console --version > fixtures/build/dir012/expected-output
+	php fixtures/build/dir012/bin/console --version > fixtures/build/dir012/expected-output
 	rm -rf fixtures/build/dir012/var/cache/prod/*
 
-	APP_ENV=prod APP_DEBUG=0 ./box compile --working-dir fixtures/build/dir012
+	php fixtures/build/dir012/bin/console cache:clear
```

Comment:
Hm I think this is an issue: IIRC the goal was to avoid to have to do a manual step before compiling hence let the warmup/cache-clear be triggered by executing a `composer install` or autoload dump which Box does

Reply:
On the line 268 the cache has been removed so I have to trigger it manually.

Btw I restored `post-install-cmd` and `post-update-cmd` scripts as they are the default.

Reply:
>On the line 268 the cache has been removed so I have to trigger it manually.

It's actually on purpose: this is to make sure that although there is no cache, whilst building the PHAR, the cache is going to be warmed up. In other words I want to make sure it's possible for the user to ship its Symfony app without having to manually dump the cache beforehand

Reply:
I added `post-autoload-cmd` back again and the most strange thing is it works on PHP 7.4 but fails on PHP 7.2 and 7.3.
Locally I can't reproduce it.